### PR TITLE
add two react js file to frontend 

### PR DESCRIPTION
**Pull request by ShuoGH**
add two file and connect the front end to the backend, to realize the signing up and logging in functions.

### CODE REVIEW

**Comment by ShuoGH** [comment]
oh sorry, not only two js file, but also make some changes on the existing files.
screenshot 2019-01-03 at 15 38 17

File: backend/project_api/project_api/settings.py [hunk 1]
```diff
@@ -97,34 +100,34 @@
...
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.mysql',
-            'NAME': 'cookingbook',
-            'USER': 'root',  #NEED A NEW USER, this was for testing only
-            'PASSWORD': 'cookingbook',
-            'HOST': '0.0.0.0',  #ip of mysql server
-            'PORT': '3306',
-        }
+# databaseDSN = os.environ.get('COOKING_APP_DATABASE_DSN', 'mysql://root:cookingbook@tcp(0.0.0.0:3306)/cookingbook')  #mysql://root:cookingbook@tcp(localhost:3306)/cookingbook
```
Comment (tfadeyi) [inline backend/project_api/project_api/settings.py]: This part can't be commented out it sets the database params, for local development/testing, CI/CD and production.
To set database DSN and params just run: 
```export COOKING_APP_DATABASE_DSN="mysql://root:<password>@tcp(0.0.0.0:3306)/<db_name>"```

https://github.com/Group-W-Comp6244/Coursework2/blob/development/backend/getting_started.md

**Comment by tfadeyi** [comment]
in the settings.py you can't comment out the database part it sets the database params, for local development/testing, CI/CD and production.
To set database DSN and params just run:
export COOKING_APP_DATABASE_DSN="mysql://root:<password>@tcp(localhost:3306)/<db_name>"

https://github.com/Group-W-Comp6244/Coursework2/blob/development/backend/getting_started.md

**Comment by tfadeyi** [comment]
Also did u install any new dependency?
If so need to add the dependency to both the requirements.txt:
https://github.com/Group-W-Comp6244/Coursework2/blob/development/backend/requirements.txt
https://github.com/Group-W-Comp6244/Coursework2/blob/development/backend/project_api/requirements.txt

**Pull request merged**